cmd/kured/main.go

```diff
@@ -489,6 +547,7 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 		if err != nil {
 			log.Fatalf("Error retrieving node object via k8s API: %v", err)
 		}
+
```

Comment:
Please do not add those spaces changes in this commit. If you want to change the formatting to look better for you, please do it in a different commit/PR.